fast_llm/models/ssm/config.py

```diff
@@ -0,0 +1,187 @@
+import logging
+import math
+import typing
+
+from fast_llm.config import Field, FieldHint, FieldUpdate, config_class
+from fast_llm.data.data.gpt.config import GPTDataConfig
+from fast_llm.engine.checkpoint.config import CheckpointFormat, CheckpointHandler
+from fast_llm.engine.multi_stage.config import FastLLMModelConfig, PretrainedFastLLMModelConfig
+from fast_llm.engine.training.config import TrainerConfig
+from fast_llm.layers.language_model.config import LanguageModelBaseConfig
+from fast_llm.layers.ssm.config import SSMDimNames, SSMLayerConfig
+from fast_llm.models.gpt.config import GPTArchitectureConfig
+from fast_llm.tensor import TensorDim, TensorSpace
+from fast_llm.utils import Assert
+
+if typing.TYPE_CHECKING:
+    from fast_llm.models.ssm.model import HybridSSMModel
+
+logger = logging.getLogger(__name__)
+
+
+@config_class
+class HybridSSMArchitectureConfig(GPTArchitectureConfig):
```

Comment:
This inherits lots of options, need to find out which ones are't supported and address or explicitly prevent. So far I saw MTP, not sure if there are more.

Reply:
they will all be eventually present, even MTP. let's not overcomplicate this PR with checks that will be removed a couple of weeks later.